snuba/datasets/storages/generic_metrics.py

```diff
@@ -160,7 +171,7 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
     query_processors=shared_query_processors,
 )
 
-distributions_bucket_storage = WritableTableStorage(
+distributions_bucket_storage_old = WritableTableStorage(
```

Comment:
I'd like to create a completely separate storage, like in a separate file separate. Basically we should avoid editing any existing files (except maybe the factory files).